core/src/main/resources/schema/app/20180426092948_BUG-100730_Add_state_status_column_to_flowlog_table.sql

```diff
@@ -0,0 +1,10 @@
+-- // BUG-100730 Add state status column to flowlog table
+-- Migration SQL that makes the change goes here.
+ALTER TABLE flowlog ADD COLUMN statestatus CHARACTER VARYING(255) NOT NULL DEFAULT 'SUCCESS';
```

Comment:
I think we should not use a default value here, because it will lock the whole table. The update to success should be in the next sql file. Also we should not use default values, CB must handle this.